tests/unit/executor_tests/test_build_tfds.py

```diff
@@ -104,8 +104,8 @@ def test_build_tfds_object_detection():
     train_run(config_file, expriment_id, recreate=True)
 
     # Check if the dataset was build correctly
-    train_data_num = 3
-    validation_data_num = 2
+    train_data_num = 10
+    validation_data_num = 16
```

Comment:
same as above

Reply:
This is the changes following the switching of delta-mark dataset to open_images dataset.